nuxt/components/edit/EditDraftLyrics.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    persistent
+    fullscreen
+    no-click-animation
+    hide-overlay
```

Comment:
Can you add `    transition="dialog-bottom-transition"` as a prop? The animation is inconsistent and jarring to me ;)

Reply:
Done